### PR TITLE
CHARMS Native View Subject Flags verification

### PR DESCRIPTION
**Pull request by katm007**
CHARMS Native View Subject Flags verification

### CODE REVIEW

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java [hunk 1]
```diff
@@ -9,7 +9,16 @@
 import com.nci.automation.utils.CucumberLogUtils;
 import com.nci.automation.web.CommonUtils;
 import com.nci.automation.web.JavascriptUtils;
+import com.nci.automation.web.WebDriverUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
```
Comment (sarwarahmed1) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java:17]: Remove extra line on 17

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java [hunk 2]
```diff
@@ -9,7 +9,16 @@
...
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
```
Comment (sarwarahmed1) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java:21]: Remove extra line 21

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java [hunk 3]
```diff
@@ -9,13 +9,21 @@
...
 import com.nci.automation.web.JavascriptUtils;
+import com.nci.automation.web.WebDriverUtils;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
```
Comment (sarwarahmed1) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java:20]: Remove extra line 20

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java [hunk 4]
```diff
@@ -9,13 +9,21 @@
...
 import java.io.*;
 import java.net.HttpURLConnection;
 import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
```
Comment (sarwarahmed1) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_All_Steps.java:26]: Remove extra line 26

**Review by sarwarahmed1 — CHANGES_REQUESTED** [review]

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java (hunk 1 above)
Comment (katm007) [reply src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java]: Removed all the extra lines!

**Review by puzikovam2 — APPROVED** [review]
approved by @puzikovam2

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java [hunk 5]
```diff
@@ -101,7 +105,7 @@ public static void submitParticipantForReviewAndEligibility() {
         CucumberLogUtils.logScreenshot();
         CommonUtils.clickOnElement(nativeViewCHARMSParticipantDetailsPage.nativeViewPatientDetailsSubmitForEligibilityButton);
         CommonUtils.sleep(1500);
-        softAssert.assertEquals(nativeViewCHARMSParticipantDetailsPage.nativeViewPatientDetailsSubjectID.getText().split("-")[0] , nativeViewCHARMSParticipantDetailsPage.nativeViewPatientDetailsFamilyID.getText());
+        softAssert.assertEquals(nativeViewCHARMSParticipantDetailsPage.nativeViewPatientDetailsSubjectID.getText().split("-")[0], nativeViewCHARMSParticipantDetailsPage.nativeViewPatientDetailsFamilyID.getText());
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java:108]: Could you use the softAssert from the Hooks class across this Class? Lines 108, 114, 115, 276, 277, 287, 288, 289, 404.

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
please review the comment.

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java (hunk 5 above)
Comment (katm007) [reply src/test/java/ServiceNow/CHARMS/Steps/RAS_Common_Methods.java:108]: This is the softAssert from the Hooks class

**Review by sarwarahmed1 — APPROVED** [review]

**Pull request merged**